app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -4,9 +4,10 @@ heading: "Support training to teach if you're disabled"
 description: |-
     Find out about the support you can get while training to teach if you're disabled.
 related_content:
-    Who do you want to teach?: "/train-to-be-a-teacher/who-do-you-want-to-teach"
     Becoming a teacher with a hearing impairment: "/blog/overcoming-challenges-to-become-a-teacher-hearing-impairment"
     Becoming a teacher with autism: "/blog/overcoming-challenges-to-become-a-teacher-autism"
+    Is teaching right for me? : "/is-teaching-right-for-me"
```

Comment:
Is this a bit jarring on this page @EllieNodder? Does it sound like teaching might not be right for people with disabilities (which is not at all what we want to say)?